infrastructure/src/main/java/org/corfudb/infrastructure/ServerHandshakeHandler.java

```diff
@@ -183,7 +183,6 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause)
             if (!this.state.completed()) {
                 log.error("exceptionCaught: Handshake timeout checker: timed out. Close Connection.");
                 this.state.set(true, false);
-                ctx.channel().close();
```

Comment:
Not sure if I completely understand this. If the handshake fail, why leave the channel open?

Reply:
@Maithem We will called channel close() at the end of this exceptionCaught() method (not showing up in the review view).